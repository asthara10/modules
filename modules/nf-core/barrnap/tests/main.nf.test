nextflow_process {

    name "Test Process BARRNAP"
    script "../main.nf"
    process "BARRNAP"

    tag "modules"
    tag "modules_nfcore"
    tag "barrnap"

    test("test_barrnap") {


        when {
            process {
                """
                
                input[0] = [
        [ id:'test', single_end:false ], // meta map
        file(params.test_data['sarscov2']['illumina']['contigs_fasta'], checkIfExists: true)
    ]

                """
            }
        }

        then {
            assertAll(
                { assert process.success },
                { assert snapshot(process.out.versions).match("versions") }
                { assert snapshot(process.out).match() }
            )
        }

    }

}
