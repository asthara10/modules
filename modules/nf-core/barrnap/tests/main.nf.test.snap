{
    "versions": {
        "content": [
            [
                "versions.yml:md5,447eafa60f76f6b84d1c41a2f5c2f76b"
            ]
        ],
        "timestamp": "2023-11-24T14:40:39.449387"
    }
}